<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMain</class>
 <widget class="QMainWindow" name="winMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>470</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Linux Launcher</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblRealm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Game server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Realm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLanguage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>99</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAccount">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>138</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>174</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboRealm">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>55</y>
      <width>201</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select game server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboLanguage">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>94</y>
      <width>260</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select game language&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAccount">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>134</y>
      <width>260</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>172</y>
      <width>260</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="imgMain">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>55</y>
      <width>401</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>209</y>
      <width>90</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start your adventure!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSaveSettings">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>211</y>
      <width>260</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Remember these settings</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtStatus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>248</y>
      <width>385</width>
      <height>213</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbout">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;About&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtFeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinimize">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimize&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSwitchGame">
    <property name="geometry">
     <rect>
      <x>733</x>
      <y>60</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switch game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <action name="actionHideWinMain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Main Window</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPatch">
   <property name="text">
    <string>Patch</string>
   </property>
   <property name="iconText">
    <string>Patch</string>
   </property>
   <property name="toolTip">
    <string>Patch</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDDOTest">
   <property name="text">
    <string>Dungeons &amp; Dragons Online (Test)</string>
   </property>
  </action>
  <action name="actionLOTROTest">
   <property name="text">
    <string>Lord of the Rings Online (Test)</string>
   </property>
  </action>
  <action name="actionLOTRO">
   <property name="text">
    <string>Lord of the Rings Online</string>
   </property>
  </action>
  <action name="actionDDO">
   <property name="text">
    <string>Dungeions &amp; Dragons Online</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
